official/projects/cots_detector/crown_of_thorns_starfish_detection_pipeline.ipynb

```diff
@@ -64,7 +64,7 @@
     "id": "jDiIX2xawkJw"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        def __init__(self, tid=1, ft=3.0, iou=0.5, tt=2.0, bb=32, of_params=None):
</pre><p>Not one for this change but it feels a bit inconsistent that everything else is a dataclass, but this class sets up the members by hand.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/models/pull/10640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>Yeah, the main reason I skipped dataclasses here was that this class does a bunch of self-modification. And to me, that feels the opposite of data-classy.</p>